testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/testcontainers/PortBinder.java

```diff
@@ -33,12 +32,17 @@ public PortBinder(EnvironmentOptions environmentOptions)
 
     public void exposePort(DockerContainer container, int port)
     {
+        container.addExposedPort(port); // Still export port, at a random free number, as certain startup checks require this.
         if (bindPorts) {
-            ContainerUtil.exposePort(container, port);
-        }
-        else {
-            // Still export port, at a random free number, as certain startup checks require this.
-            container.addExposedPort(port);
+            container.withFixedExposedPort(port, port);
         }
     }
+
+    // This method exposed port unconditionally on the host machine.
+    // Usage should be limited to exposing debugging ports.
+    public static void unsafelyExposePort(DockerContainer container, int port)
```

Comment:
unsafely -> unconditionally?

Reply:
i proposed to add "unsafely" to clearly indicate it's not _safe_ to _unconditionally_ call this method (no pun intended)